core/os/os2/file.odin

```diff
@@ -46,10 +46,9 @@ O_SPARSE  :: File_Flags{.Sparse}
 O_CLOEXEC :: File_Flags{.Close_On_Exec}
 
 
-
-stdin:  ^File = nil // OS-Specific
-stdout: ^File = nil // OS-Specific
-stderr: ^File = nil // OS-Specific
+stdin:  ^File = &_stdin
+stdout: ^File = &_stdout
+stderr: ^File = &_stderr
```

Comment:
If you see in file_windows.odin, `@(init)` and `@(fini)` calls are used initialize these variables. I would not rely on the global variables, especially since they will not be closed on (successful) program exit.

Reply:
Also handled in new commit.